vividus-tests/src/main/resources/properties/suite/system/mobile_app/ios/suite.properties

```diff
@@ -1,3 +1,3 @@
 bdd.batch-1.meta-filters=groovy: !targetPlatform || targetPlatform.contains('ios')
 
-bdd.story-loader.batch-1.resource-include-patterns=MobileAppStepsTests.story,VisualTesting.story
+bdd.story-loader.batch-1.resource-include-patterns=MobileAppStepsTests.story,VisualTesting.story,MobileDprCheck.story
```

Comment:
MobileDprCheck.story - I think we should add some IOS part to this story name since this is purely for iOS

Reply:
what about `MobileIOSDprCheck.story`

Reply:
done